src/main/java/de/dennisguse/opentracks/Startup.java

```diff
@@ -37,6 +43,40 @@ public void onCreate() {
         }
     }
 
+    @Override
+    protected void attachBaseContext(final Context base) {
+        super.attachBaseContext(base);
+
+        // we don't want to handle crashes occurring inside crash reporter activity/process;
+        // let the platform deal with those
+        boolean isCrashReportingProcess = getAppProcessName().endsWith(":crash");
+
+        if (!isCrashReportingProcess) {
+            Thread.UncaughtExceptionHandler defaultPlatformHandler = Thread.getDefaultUncaughtExceptionHandler();
+            ExceptionHandler crashReporter = new ExceptionHandler(this,
+                    defaultPlatformHandler);
+            Thread.setDefaultUncaughtExceptionHandler(crashReporter);
+        }
+    }
+
+    private String getAppProcessName() {
```

Comment:
I would avoid this; there is a good reason why it was removed as of Android P.

Reply:
Hmm, good point. I found another workaround for older devices here: https://stackoverflow.com/a/55842542/4952072
Better or should we omit this check completely for older devices? I don't know what can happen, if an Exception is happening in the ErrorHandler itself.

Reply:
Actually, what is the benefit of exposing this?
Will this give us some meaningful intel?

Reply:
What do you mean by "of exposing this"?
As I understand the ShowErrorActivity is launched with its own process name `:crash` as defined in the manifest: https://github.com/OpenTracksApp/OpenTracks/pull/1237/files#diff-6305bbf50570f0946de077f04e8ea56c53d8f6e1b53ec403c8c2ddb10d531db1R373
This is used to distinguish between the normal app run and the crash report. If the crash report is launched then the `DefaultUncaughtExceptionHandler` is not changed to prevent a loop in case there is an Exception in the `ExceptionHandler` or `ShowErrorActivity`. 
Do you understand it differently? Do I miss something?

Reply:
Now I got ;)
Then my suggestion is to refactor getAppPackageName() to isCrashReportingProcess() and do the whole thing in there.

Reply:
Ah ok, the method name is not clear. Yes, that can be improved. I've moved the `processName.endsWith(":crash")` logic into the `getAppPackageName()` method and renamed it to `isCrashReportingProcess()`, because it is only used for that. Looks better now, what do you think?

Reply:
Awesome!